python/Dockerfile

```diff
@@ -26,7 +27,7 @@ RUN apt-get update \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/* \
   && cd /tmp \
-  && wget https://www.python.org/ftp/python/3.12.0/Python-3.12.0.tgz \
+  && wget -q https://www.python.org/ftp/python/3.12.0/Python-3.12.0.tgz \
```

Comment:
@ivanovmg it's better to use `--quiet`, see: https://matklad.github.io/2025/03/21/use-long-options-in-scripts.html